### PR TITLE
Update Docs

### LINKED ISSUE #55: Update docs for this theme
We have changed a lot about how this theme works and will need to update the README and documentation for this theme itself.

### PR DESCRIPTION
**Pull request by charnpreetsingh**
Added in `development` and user install `instructions`
Closes #55

### CODE REVIEW

File: docs/source/development.rst [hunk 1]
```diff
@@ -28,3 +28,53 @@ server and navigating to 0.0.0.0:8000 in your web browser.
...
+.. code::
+
+    npm run css
+
+
+Modifying Sphinx templates:
+
+1. Make appropriate changes to templates.
+
+2. Uninstall ``jupyter_alabaster_theme``:
```
Comment (willingc) [inline docs/source/development.rst:63]: Do you have to uninstall?

Comment (charnpreetsingh) [reply docs/source/development.rst:63]: I'm not exactly sure. When I did development locally, I found that just doing `pip install .` didn't change the package

File: docs/source/user.rst [hunk 2]
```diff
@@ -6,16 +6,51 @@ User
...
 .. code-block:: python
 
-    # At the top
-    from jupyter_alabaster_theme import *
-    init_theme()
-    # ...
-    # Comment out the `html_theme =` line
+    if not on_rtd:
+       # only import and set the theme if we're building docs locally
+       import sphinx_rtd_theme
```
Comment (willingc) [inline docs/source/user.rst:41]: Is there a reason to use sphinx_rtd_theme? If building locally, you can still use jupyter_alabaster_theme.

**Review by willingc — CHANGES_REQUESTED** [review]
A few questions but otherwise looks good. I would recommend changing the local build to use the new theme.

File: docs/source/user.rst (hunk 2 above)
Comment (charnpreetsingh) [reply docs/source/user.rst:41]: @willingc That section is the code block that needs to be removed if present

**Pull request merged**